Automattic/wp-calypso
# e2e: Support increasing number of parallel tests to improve run times

#### Changes proposed in this Pull Request
* There are some changes to packages and run.sh that will allow for splitting tests across containers (for use in wp-e2e-tests-for-branches)

#### Testing instructions
* Try using `./run.sh -g -f spec/wp-signup-spec.js,spec/wp-reader-spec.js' locally and make sure that only tests from those two files run.
